content/boilerplates/notes/1.2.1.md

```diff
@@ -0,0 +1,16 @@
+## Bug fixes and minor enhancements
+
+- Fix duplicate CRD being generated in the install ([Issue 14976](https://github.com/istio/istio/issues/14976))
+- Allow users to disable istio default retries by setting retries to 0 ([Issue 14900](https://github.com/istio/istio/issues/14900))
+- Improve citadel log ([Pull Request 15032](https://github.com/istio/istio/pull/15032))
+- Add citadel replicaCount flag back and workloadCertTtl to support multi-citadel and short cert TTL for perf test ([Pull Request 15066](https://github.com/istio/istio/pull/15066))
+- Fix Mixer CRD store issue ([Issue 14841](https://github.com/istio/istio/issues/14841))
```

Comment:
I would perfer a user related msg like "Fix Mixer unable to start when Galley is disabled"

Reply:
```suggestion
- Fix Mixer unable to start when Galley is disabled ([Issue 14841](https://github.com/istio/istio/issues/14841))
```